### ISSUE TITLE
Pint Support not working (even with older version)

### ISSUE THREAD

**Issue by Akecel** [shivammathur/setup-php#647]
**Describe the bug**
I had already posted a comment in another issue already closed for this problem (#613). And I saw that you had made PRs on the side of the `laravel/pint` repository to correct a problem making that the `.phar` was not available in the lasts releases (thank you very much for the reactivity by the way).

Having discussed with the person in charge of the Pint releases, I know that no new release is planned for the moment and that therefore, I could not simply wait for a new one. So I decided to use the last release with a `.phar` (`1.1.0`), unfortunately it is still impossible to setup Pint in my workflow even though the` .phar` seems to be present in this release

```
     - name: Setup PHP
        uses: shivammathur/setup-php@v1
        with:
          php-version: '8.0'
          tools: pint:1.1.0
      - name: Run Laravel Pint without Fix
        run: pint --test
```

```
| ==> Setup Tools
| /usr/bin/php: error while loading shared libraries: libsodium.so.23: cannot open shared object file: No such file or directory
| ✓ composer Added composer 2.4.2
| /usr/bin/chmod: cannot access '/usr/local/bin/1.1.0/pint.phar': No such file or directory
| ✗ 1.1.0/pint.phar Could not setup 1.1.0/pint.phar
```

**Version**
- [x] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.

**Runners**

- [x] GitHub Hosted

**Operating systems**
Ubuntu Latest

**PHP versions**
I tested with PHP 8.0 and PHP 8.1

**To Reproduce**
```
jobs:
  lint:
    name: Lint
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Setup PHP
        uses: shivammathur/setup-php@v1
        with:
          php-version: '8.0'
          tools: pint:1.1.0
      - name: Run Laravel Pint without Fix
        run: pint --test
```

**Expected behavior**

Error during setup : 
```
| ==> Setup Tools
| /usr/bin/php: error while loading shared libraries: libsodium.so.23: cannot open shared object file: No such file or directory
| ✓ composer Added composer 2.4.2
| /usr/bin/chmod: cannot access '/usr/local/bin/1.1.0/pint.phar': No such file or directory
| ✗ 1.1.0/pint.phar Could not setup 1.1.0/pint.phar
```

**Additional context**
* My first comment can be found in this issue (#613)
* Last Pint release with .phar ([https://github.com/laravel/pint/releases/tag/v1.1.0](https://github.com/laravel/pint/releases/tag/v1.1.0))

**Comment by shivammathur** [comment]
Version support fixed in https://github.com/shivammathur/setup-php/commit/a49dfe8457df0b95e6d418402c287f60a40695b4
Test workflow: https://github.com/shivammathur/test-setup-php/actions/runs/3127180418/workflow

It will be in `v2` in the next release

**Comment by Akecel** [comment]
Thanks a lot !

**Comment by shivammathur** [comment]
Released [2.22.0](https://github.com/shivammathur/setup-php/releases/tag/2.22.0) with the fix.
